spec/scene-perception.html

```diff
@@ -218,6 +218,18 @@
             Allows user to get length of side of voxel cube in meters.
           </dd>
           <dt>
+            Promise&lt;Intrinsics&gt; getInternalCameraIntrinsics()
```

Comment:
I can't fine the API spec for `Intrinsics`, did I miss it? And I prefer to use `CameraIntrinsics` name.